Core/include/Acts/Utilities/BinningData.hpp

```diff
@@ -41,13 +42,13 @@ namespace Acts {
 ///
 class BinningData {
  public:
-  BinningType type{};       ///< binning type: equidistant, arbitrary
-  BinningOption option{};   ///< binning option: open, closed
-  BinningValue binvalue{};  ///< binning value: binX, binY, binZ, binR ...
-  float min{};              ///< minimum value
-  float max{};              ///< maximum value
-  float step{};             ///< binning step
-  bool zdim{};              ///< zero dimensional binning : direct access
+  BinningType type{};        ///< binning type: equidistant, arbitrary
+  BinningOption option{};    ///< binning option: open, closed
+  AxisDirection binvalue{};  ///< axis direction: AxisX, AxisY, AxisZ, ...
```

Comment:
This stays `binvalue`?

Reply:
Yes, I will not touch the BinUtility further - we will phase it out ... the changes necessary are only short-lived, so I am not sure it is actually worth it.